chia/wallet/util/puzzle_compression.py

```diff
@@ -35,6 +35,7 @@
     + bytes(NFT_TRANSFER_PROGRAM_DEFAULT),
     bytes(CAT_MOD),
     bytes(OFFER_MOD),
+    b"break",  # purposefully break compatibility with older versions
```

Comment:
a minor suggestion I had was to make this an empty string. That way we won't be feeding any more (arbitrary) bytes into the zdict (potentially reducing compression ratio). It would also let use make this quirk disappear in the future, when we no longer support old versions